Frameworks/include/CoreGraphics/DWriteWrapper.h

```diff
@@ -75,4 +76,9 @@ COREGRAPHICS_EXPORT HRESULT _DWriteFontGetBoundingBoxesForGlyphs(
 // DWrite functions relating to font binary data
 COREGRAPHICS_EXPORT HRESULT _DWriteRegisterFontsWithDatas(CFArrayRef fontDatas, CFArrayRef* errors);
 COREGRAPHICS_EXPORT HRESULT _DWriteUnregisterFontsWithDatas(CFArrayRef fontDatas, CFArrayRef* errors);
-COREGRAPHICS_EXPORT HRESULT _DWriteCreateFontFaceWithData(CGDataProviderRef data, IDWriteFontFace** outFontFace);
\ No newline at end of file
+COREGRAPHICS_EXPORT HRESULT _DWriteCreateFontFaceWithData(CGDataProviderRef data, IDWriteFontFace** outFontFace);
+
+inline uint32_t _CTToDWriteFontTableTag(uint32_t tag) {
+    // CT has the opposite byte order of DWrite, so we need 'BASE' -> 'ESAB'
```

Comment:
I can see that case where someone would just use 'BASE' or something.  The values are public and the conversion is cheap so I think it's worth it to just do the conversion "Just in case"